src/lib/utils/thorchain/hooks/useSendThorTx.tsx

```diff
@@ -333,6 +335,8 @@ export const useSendThorTx = ({
             wallet,
           })
 
+          await checkLedgerAppOpenIfLedgerConnected(asset.chainId)
```

Comment:
preferably-blocking: seems like this line is not handling every cases (try to deposit into runepool for example, or maybe even THORChain BTC Savers)

Reply:
Nice catch ser, blaming jetlag on this one. 18ee336679